pom.xml

```diff
@@ -133,7 +133,7 @@
     <dependency>
       <groupId>com.spotify</groupId>
       <artifactId>docker-client</artifactId>
-      <version>8.11.4</version>
+      <version>8.14.1-SNAPSHOT</version>
```

Comment:
Requires this [docker-client PR](https://github.com/spotify/docker-client/pull/1093) to be merged and released.

Reply:
whats the relation of that PR to this one?

Reply:
I think you can change this to use 8.14.0 to uncouple the two chanes

Reply:
I was testing docker-maven-plugin locally and ran into a docker-client bug there.